posts/Davide/2018-05-25-hibernate-ogm-5-4-Beta1-released.adoc

```diff
@@ -0,0 +1,83 @@
+= "Hibernate OGM 5.4.0.Beta1 release"
+Davide D'Alto
+:awestruct-tags: [ "Hibernate OGM", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+http://hibernate.org/ogm/releases/5.4/#get-it[Hibernate OGM 5.4.0.Beta1] has been released!
+
+The main differences from the previous release is the upgrade to Hibernate ORM 5.3.0.Final, 
+the support for Infinispan Remote native queries and the use of cluster counters
```

Comment:
JPQL and native. I think not just native

Reply:
I mentioned it with more details later.

Technically you could query using JPQL even before, it's the way we do it that's changed.

Reply:
I will add `JPQL (without using Hibernate Search)`